tests/Block/ActionsUnitTest.php

```diff
@@ -5,9 +5,8 @@
 use Maknz\Slack\Block\Actions;
 use Maknz\Slack\BlockElement\StaticSelect;
 use Maknz\Slack\BlockElement\Text;
-use Slack\Tests\TestCase;
 
-class ActionsUnitTest extends TestCase
+class ActionsUnitTest extends \PHPUnit\Framework\TestCase
```

Comment:
these changes are unnecessary

Reply:
I reverted all of this.  There is one warning message that TeestCase has no tests, but that can be ignored.